### DISCUSSION TITLE
date_trunc 函数无法使用

### DISCUSSION THREAD

**Discussion by oliverzzz66** [apache/doris#17029] [A - General / Q&A] [answered]
![image](https://user-images.githubusercontent.com/67142134/220580681-4875d049-9425-4548-af36-16f57434a6d7.png)
doris版本为：1.1.3-rc02

**Comment by zy-kkk** [comment, marked as answer]
1.1系列并未支持这个函数，请升级1.2系列使用

**Reply by oliverzzz66** [reply]
tks
